VKClientRM/VKClientRM/Controllers/GroupUserTableViewController.swift

```diff
@@ -2,5 +2,92 @@
 // Copyright © RoadMap. All rights reserved.
 
 import UIKit
-/// экран с группами пользователей
-final class GroupUserTableViewController: UITableViewController {}
+/// Экран с группами пользователя
+final class GroupUserTableViewController: UITableViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let segueID = "GoToSearchGroupTableVC"
+        static let groupUserCellID = "GroupUserCell"
+        static let addGroupActionSegueID = "AddGroupAction"
+        static let groupUserPhotoOneName = "FriendPhotoOne"
+        static let groupUserPhotoSecondName = "FriendPhotoSecond"
+        static let groupUserPhotoThirdName = "FriendPhotoThird"
+        static let groupUserNameOneName = "SwiftDevelop"
+        static let groupUserNameSecondName = "Программирование"
+        static let groupUserNameThirdName = "Спартак"
+        static let groupUserNameFourName = "Футбол"
+        static let groupUserNameFiveName = "Спорт"
+        static let groupUserNameSixName = "Тай"
+    }
+
+    // MARK: - Private Properties
+
+    private var allGroups = [
+        Group(groupName: Constants.groupUserNameFourName, groupPhotoName: Constants.groupUserPhotoOneName),
+        Group(groupName: Constants.groupUserNameFiveName, groupPhotoName: Constants.groupUserPhotoSecondName),
+        Group(groupName: Constants.groupUserNameSixName, groupPhotoName: Constants.groupUserPhotoThirdName)
+    ]
+
+    private var userGroups = [
+        Group(groupName: Constants.groupUserNameOneName, groupPhotoName: Constants.groupUserPhotoOneName),
+        Group(groupName: Constants.groupUserNameSecondName, groupPhotoName: Constants.groupUserPhotoSecondName),
+        Group(groupName: Constants.groupUserNameThirdName, groupPhotoName: Constants.groupUserPhotoThirdName)
+    ]
+
+    // MARK: - IBAction
+
+    @IBAction private func addGroupAction(segue: UIStoryboardSegue) {
+        guard
+            segue.identifier == Constants.addGroupActionSegueID,
+            let source = segue.source as? SearchGroupTableViewController,
+            let indexPath = source.tableView.indexPathForSelectedRow,
+            let group = source.returnGroup(index: indexPath.row),
+            !userGroups.contains(group)
+        else { return }
+        for (index, groupFromAllGroups) in allGroups.enumerated() {
+            guard group == groupFromAllGroups else { continue }
+            allGroups.remove(at: index)
+        }
+        userGroups.append(group)
+        tableView.reloadData()
+    }
+
+    // MARK: - Public Methods
```

Comment:
```suggestion
    // MARK: - Public Methods
     // MARK: - IBAction
```
в такой последовательности располагаем